setup.py

```diff
@@ -51,7 +51,12 @@
               'snapcraft.plugins',
               'snapcraft.storeapi'],
     package_data={'snapcraft.internal.repo': ['manifest.txt']},
-    scripts=['bin/snapcraft', 'bin/snapcraft-parser'],
+    entry_points={
+        'console_scripts': [
+            'snapcraft = snapcraft.cli.__main__:run',
+            'snapcraft-parser = snapcraft.internal.parser:main',
+        ],
+    },
```

Comment:
@kalikiana this is just standard python, it is a portable way to deliver.
@elopio not ready to delete the scripts in `bin` until we say we don't care about local development anymore.